docker.sh

```diff
@@ -1,39 +1,55 @@
 #!/bin/bash
-set -eo pipefail
+set -euox pipefail
 
-PARAM=${1:-push};
+NAME="indexd"
+PARAM=${1:-DO_NOT_PUSH};
 
-IMAGE_NAME="${DOCKER_RELEASE_REGISTRY:=quay.io}/ncigdc/indexd"
+BASE_CONTAINER_VERSION=1.2.0  # upgrade later
 
-# avoid installing git
-COMMIT=$(git rev-parse HEAD) && echo "COMMIT=\"${COMMIT}\"" >indexd/index/version_data.py
+IMAGE_NAME="${DOCKER_RELEASE_REGISTRY:=quay.io}/ncigdc/${NAME}"
```

Comment:
Outside the context of this PR:

At some point, we'll be deprecating quay ... no?
i.e. we won't use it because we no longer use travis and all artifacts and docker imgs can live in nexus.

Do we have a JIRA for doing all that cleanup soon?

Reply:
For this one, I am not sure whether we are retiring quay and travis. 
I think we need a discussion about it.